src/org/thoughtcrime/securesms/service/RegistrationService.java

```diff
@@ -260,17 +260,16 @@ private void handleCommonRegistration(TextSecureAccountManager accountManager, S
     DatabaseFactory.getIdentityDatabase(this).saveIdentity(self.getRecipientId(), identityKey.getPublicKey());
     DirectoryHelper.refreshDirectory(this, accountManager, number);
 
-    //if (GooglePlayServicesUtil.isGooglePlayServicesAvailable(this) == ConnectionResult.SUCCESS &&
-    //    !BuildConfig.FORCE_WEBSOCKETS)
-    //{
-    //  RedPhoneAccountManager redPhoneAccountManager = new RedPhoneAccountManager(BuildConfig.REDPHONE_MASTER_URL,
-    //                                                                           new RedPhoneTrustStore(this),
-    //                                                                           number, password);
-    //  String verificationToken = accountManager.getAccountVerificationToken();
-    //  redPhoneAccountManager.createAccount(verificationToken, new RedPhoneAccountAttributes(signalingKey,
-    //          TextSecurePreferences.getGcmRegistrationId(this)));
-    //}
-    accountManager.getAccountVerificationToken();
+    String verificationToken = accountManager.getAccountVerificationToken();
```

Comment:
> I guess you are allowing RedPhone, when using GCM and still disabling it, if WebSocket is enabled?

Yes, this is behavior I want to achieve. But this whole patch is not included in my builds on https://fdroid.eutopia.cz yet (because I am building it directly from this JavaJens repository).